src/mixer/audio.ts

```diff
@@ -415,22 +448,40 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
       .connect(this.micFinalAnalyser)
       .connect(this.streamingAnalyser);
 
-    // Send the final compressor (all players and guests) to the headphones.
-    this.finalCompressor.connect(this.audioContext.destination);
-
-    // Also send the final compressor to the streaming analyser on to the stream.
-    this.finalCompressor.connect(this.streamingAnalyser);
+    this._connectFinalCompressor(true);
 
     // Send the streaming analyser to the Streamer!
     this.streamingAnalyser.connect(this.streamingDestination);
 
     // Feed the news in/out reminders to the headphones too.
     this.newsStartCountdownNode.connect(this.audioContext.destination);
     this.newsEndCountdownNode.connect(this.audioContext.destination);
+
+    // Send the headphones feed to the headphones.
+    const db = -12; // DB gain on headphones (-6 to match default trim)
+    this.headphonesNode.gain.value = Math.pow(10, db / 20);
+    this.headphonesNode.connect(this.audioContext.destination);
+  }
+
+  _connectFinalCompressor(headphones: boolean) {
```

Comment:
`headphones` isn't really clear on what it does - please either rename it or add a doc comment explaining its semantics

Reply:
Did both 👍